src/robot_dart/gui/magnum/glfw_application.cpp

```diff
@@ -18,6 +19,7 @@ namespace robot_dart {
                 conf.setSize({static_cast<int>(width), static_cast<int>(height)});
                 conf.setWindowFlags(Configuration::WindowFlag::Resizable);
                 glConf.setSampleCount(8);
+                glConf.setVersion(Magnum::GL::Version::GL320);
```

Comment:
I'm pretty sure this will cause NVidia drivers to be stuck at GL 3.2 (instead of going all the way to 4.6). Why was it needed? :)

Reply:
This is indeed true. But all we need is GL 3.2 + one additional feature. Will it affect performance if we do not use GL 4.6 (when available)?

Reply:
No, but at some point you might want to opt-in into fancier features available only on newer versions (extensions depending on GL 4.0+ and such, for example). Right now there won't be any difference I think.